.github/workflows/news_scraper.yml

```diff
@@ -0,0 +1,29 @@
+name: Run News Scraper Daily at 4 PM MST
+
+on:
+  schedule:
+    # 16:00:00 GMT-0600 (Mountain Daylight Time)
+    - cron: '0 22 * * *'
+  workflow_dispatch:
+
+jobs:
+  run-python-script:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install Scrapy
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+
+    - name: Run Python script
+      run: python -m scrapy crawl lwr_spider_featured -o featured.json
```

Comment:
command to run the `lwr_spider` web crawler and output the scraped data to `featured.json`